ios/Classes/PTFlutterDocumentController.h

```diff
@@ -15,6 +16,11 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic) BOOL documentLoaded;
 
 // viewer options
+@property (nonatomic, copy, nullable) NSArray<NSString *> * annotationToolbars;
+@property (nonatomic, copy, nullable) NSArray<NSString *> * hideDefaultAnnotationToolbars;
+@property (nonatomic, assign) BOOL hideAnnotationToolbarSwitcher;
+@property (nonatomic, assign) BOOL hideTopToolbars;
+@property (nonatomic, assign) BOOL hideTopAppNavBar;
```

Comment:
These are states but are named as actions. Please rename to `topAppNavBarHidden` etc.

Reply:
Well, to keep the consistency with RN? Since they are named in that format from there. Or I could update both.

Reply:
It's a weird property because it's strange to read from it. What does it mean to read `hideTopToolbars`? A bool contains a state, but this one's name is an action. I think these are probably bad names in RN too, but let's feel we need to repeat its mistakes.